test/aria/utils/ArrayTestCase.js

```diff
@@ -237,6 +237,46 @@ Aria.classDefinition({
                 result += value;
             }, this);
             this.assertTrue(result === "123459");
+        },
+
+        testEvery : function () {
+            // shortcut
+            var every = aria.utils.Array.every;
+
+            // doing stuff on empty array should do nothing
+            var count = 0;
+            every([], function () {
+                count++;
+            });
+            this.assertTrue(count === 0);
+
+            // forEach simple test
```

Comment:
I just posted my review when you did your last changes I think. This is a bit out of date, but still I think repeating the name of the tested function (`every`) all the time is not necessary.